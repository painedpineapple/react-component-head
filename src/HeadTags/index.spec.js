import React from 'react'
//
import { render } from '../../utils/testing/components'
import Meta from './'

describe('Meta Component Test', () => {
  it('Renders', () => {
    const { container } = renderSetup()

    expect(container.firstChild).toMatchSnapshot()
  })
})

function renderSetup(overrides) {
  const props = {
    ...overrides,
  }

  const wrapper = render(<Meta {...props} />)

  return {
    ...wrapper,
    props,
  }
}
