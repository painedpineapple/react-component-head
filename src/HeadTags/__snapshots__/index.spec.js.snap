https://uploads.codesandbox.io/uploads/user/2ed66950-07af-4391-8070-a6aa9e07d14c/_bhw-index.spec.js.snap